### DISCUSSION TITLE
Noise Detector Suggestion

### DISCUSSION THREAD

**Discussion by cemox35** [asteroid-team/asteroid#604] [Q&A]
Hello there,

It might be irrelevant but I want to do the Noise Detection Model in Asteroid or Pytorch Lightning that integrates with any model in Predefined Models (Conv-TasNet) in Asteroid. Any suggestion for this? Do I have to do it with the deep learning model? Thank you for all your answers.

**Comment by mpariente** [comment]
What exactly are you trying to do? What's your starting point?

**Reply by cemox35** [reply]
I just want to detect the noise (any type), just noise. If the sound is noisy, this model returns 1, if not, returns 0. It will be used in a real-time application (like that: https://www.youtube.com/watch?v=QL4HY_e21v0). I am not interested in voice, just detect like that noises.

**Reply by mpariente** [reply]
You should probably train the equivalent of a VAD model, but for noise.

**Reply by cemox35** [reply]
Do you have any suggestions for a VAD model?

**Reply by popcornell** [reply]
pyannote https://github.com/pyannote/pyannote-audio has some very good pretrained VAD models

**Reply by mpariente** [reply]
Yes, pyannote is great, and we also have one [here](https://github.com/asteroid-team/asteroid/tree/master/egs/LibriVAD).
